Cvc5Test/ApiTermWhite.lean

```diff
@@ -0,0 +1,34 @@
+import Cvc5Test.Init
```

Comment:
Fine for now. We should remove white tests at some point.

Reply:
I can remove it as this PR will be closely followed by the exception-handling and black-testing one(s), and we already have `MkTerms1.lean` and `Solver1Parse.lean` as (temporary) test examples.

Reply:
OK!